backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inout/event/InOutUserNotificationsProducer.java

```diff
@@ -189,11 +189,11 @@ public InOutUserNotificationsProducer notifyShipmentError(final String sourceInf
 
 	private void postNotification(final UserNotificationRequest notification)
 	{
-		notificationBL.sendAfterCommit(notification);
+		notificationBL.send(notification);
```

Comment:
Y not sendAfterCommit?

Reply:
when the wp fails, the trx is rolled back and with `sendAfterCommit` there is no error notification displayed.